### PR TITLE
Adding Drift V2 endpoints Part1

### PR DESCRIPTION
**Pull request by tinystarinagalaxy**

### CODE REVIEW

**Review by aspin — APPROVED** [review]

File: provider/grpc.go [hunk 1]
```diff
@@ -110,8 +110,8 @@ func (g *GRPCClient) GetRecentBlockHash(ctx context.Context) (*pb.GetRecentBlock
 }
 
 // GetOrderbook returns the requested market's orderbook (e.g. asks and bids). Set limit to 0 for all bids / asks.
-func (g *GRPCClient) GetOrderbook(ctx context.Context, market string, limit uint32, project pb.Project) (*pb.GetOrderbookResponse, error) {
-	return g.apiClient.GetOrderbook(ctx, &pb.GetOrderbookRequest{Market: market, Limit: limit, Project: project})
+func (g *GRPCClient) GetOrderbook(ctx context.Context, market string, limit uint32, metadata bool, project pb.Project) (*pb.GetOrderbookResponse, error) {
```
Comment (aspin) [inline provider/grpc.go]: this field looks like it doesn't belong. Think you missed it as you switch from design 1 => 2?

File: provider/ws.go [hunk 2]
```diff
@@ -73,9 +73,12 @@ func (w *WSClient) RecentBlockHash(ctx context.Context) (*pb.GetRecentBlockHashR
 }
 
 // GetOrderbook returns the requested market's orderbook (e.g. asks and bids). Set limit to 0 for all bids / asks.
-func (w *WSClient) GetOrderbook(ctx context.Context, market string, limit uint32, project pb.Project) (*pb.GetOrderbookResponse, error) {
+func (w *WSClient) GetOrderbook(ctx context.Context, market string, limit uint32, metadata bool, project pb.Project) (*pb.GetOrderbookResponse, error) {
```
Comment (aspin) [inline provider/ws.go]: here too

**Review by aspin — CHANGES_REQUESTED** [review]
think a field has been left in that shouldn't be

File: provider/grpc.go (hunk 1 above)
Comment (tinystarinagalaxy) [reply provider/grpc.go]: removed

File: provider/ws.go (hunk 2 above)
Comment (tinystarinagalaxy) [reply provider/ws.go]: removed

**Review by aspin — APPROVED** [review]

**Pull request merged**